mobile/src/common/types/navigation/root-navigation-param-list.type.ts

```diff
@@ -3,6 +3,7 @@ import { RootScreenName } from '~/common/enums/enums';
 type RootNavigationParamList = {
   [RootScreenName.SIGN_UP]: undefined;
   [RootScreenName.SIGN_IN]: undefined;
+  [RootScreenName.APP]: undefined;
```

Comment:
Create AppNavigationParamList and define params for all screens and use it here via NavigatorScreenParams<AppNavigationParamList>, you can read more about it in [React Navigation Documentation](https://reactnavigation.org/docs/typescript/#organizing-types)